src/components/InputField/InputField.js

```diff
@@ -220,7 +220,7 @@ class InputField extends Component {
               name={this.props.name && this.props.name}
               className={`form__field form__field--${this.props.type} ${error}`}
               required={this.props.required}
-              autocomplete={this.props.autocomplete}
+              autoComplete={this.props.autoComplete}
```

Comment:
The actual html attribute isn't camel-cased though.

Reply:
But React prefers it to be camelCase and works the same. Same thing applies for a few html properties. I have tested this.

Reply:
ok cool didn't know react prefers it

Reply:
![Screen Shot 2019-03-13 at 12 43 48](https://user-images.githubusercontent.com/27994450/54279739-bf146180-458d-11e9-81ea-65ee97470f54.png)